recipes/strling/0.1.0/meta.yaml

```diff
@@ -0,0 +1,27 @@
+{% set name = "strling" %}
```

Comment:
The recipe should be moved one directory up (it should be at `recipes/strling/meta.yaml`). Subdirectories are generally not used for recipes unless there are good reasons for keeping multiple versions around.